deployment-bundle-steps.md

```diff
@@ -0,0 +1,135 @@
+# How to setup a project with deployment to S3
+
+1. Create S3 bucket for production / staging (your-bucket-name-production & your-bucket-name-staging). (If using cloudfront, see [optional settings](#optional-settings))
```

Comment:
Could this be more strict about naming here?

In Dist. we have so many buckets we don't know what half of them are for, if you used `repo_name-staging` or `repo_name-production` then it is obvious what is using the bucket.